core/src/services/fs/backend.rs

```diff
@@ -367,15 +372,23 @@ impl Accessor for FsBackend {
             let target_path = Self::ensure_write_abs_path(&self.root, path).await?;
             let tmp_path =
                 Self::ensure_write_abs_path(atomic_write_dir, &tmp_file_of(path)).await?;
-            (target_path, Some(tmp_path))
+
+            // If the target file exists, we should append to the end of it directly.
+            if tokio::fs::try_exists(&target_path)
```

Comment:
This should happen if args.append is true.

Reply:
🥲